src/drw_base.h

```diff
@@ -87,6 +88,27 @@ enum Version {
     AC1032,       //!< AutoCAD 2018/2019/2020
 };
 
+const std::unordered_map< const char*, DRW::Version > dwgVersionStrings {
```

Comment:
wound't that  be instanciated in all .cpp files that include this header ?

Reply:
Fixed